aws/index.yaml

```diff
@@ -0,0 +1,2 @@
+- id: AWS061
```

Comment:
What are we going to use this file for now? I thought this was part of appshield but will it have a use in this repo too?

Reply:
Yes we need some index to list all ids and where the metadata located